src/backends/interflop-vprec/interflop_vprec.c

```diff
@@ -777,10 +777,10 @@ void _interflop_enter_function(interflop_function_stack_t *stack, void *context,
     function_inst->isIntrinsicFunction = function_info->isIntrinsicFunction;
     function_inst->useFloat = function_info->useFloat;
     function_inst->useDouble = function_info->useDouble;
-    function_inst->OpsRange64 = VPREC_RANGE_BINARY64_DEFAULT;
-    function_inst->OpsPrec64 = VPREC_PRECISION_BINARY64_DEFAULT;
-    function_inst->OpsRange32 = VPREC_RANGE_BINARY32_DEFAULT;
-    function_inst->OpsPrec32 = VPREC_PRECISION_BINARY32_DEFAULT;
+    function_inst->OpsRange64 = VPRECLIB_BINARY64_RANGE;
```

Comment:
I don't think this is right.
We're building a profile of the calls, at this point.

So the default precision to be used in this function is that given by the code, not chosen by the user at a specific execution.

Reply:
Fair enough.

Reply:
I think there needs to be a more abstract way of handling vprec runs.  If I, as a user, pass in --range-binary64=11 for the virtual precision backend, I would expect that all floating points would be intercepted with this, regardless of whether an inclusive/exclusive file was passed in.

Reply:
In addition, interflop_init already sets the floating point precision at default modes:
  
   https://github.com/verificarlo/verificarlo/blob/1c9f85669a0011a32bca520c2b9972a7479a9bf1/src/backends/interflop-vprec/interflop_vprec.c#L1506

I stand correct in my commit, as it works in my case and what I expect from Verificarlo's vprec backend.